.circleci/chain_on_pr.sh

```diff
@@ -51,6 +51,7 @@ then
 
     echo "Running the E2E suite against the newly deployed isolated chain for this PR.."
     export API_ENDPOINT=http://$TESTNET_NODE_IP/vchains/$PR_CHAIN_ID/ \
+        REMOTE_ENV="true" \
```

Comment:
You don't have to set it here, `REMOTE_ENV` is the opposite of `shouldBootstrap` from `test/e2e/harness.go`.

You don't have to read it from the environment.

Reply:
Actually `shouldBootstrap` isn't exactly the same as it is identifying the Docker e2e as a remote environment whereas this is probably the ONLY place you don't want to skip this test (that sets up the pre-historic blocks data)

That's why I introduced this new variable and new logic inside the e2e harness.